services/common/boltzmann/client/index.js

```diff
@@ -185,9 +300,10 @@ module.exports = class Client {
         message = parsed.message || body
         code = parsed.code || code
       } catch {
+        ;
```

Comment:
I'm sure you're probably working on this now, but commenting just in case. Empty line / no caught error